src/state/locale/types.js

```diff
@@ -0,0 +1,3 @@
+
+// eslint-disable-next-line
```

Comment:
always specify which rule has been disabled. avoid blank lines at the beginning as well